src/components/Chat/constants.ts

```diff
@@ -1,24 +1,7 @@
-import dfinityimg from "images/avatar/01.png";
-import ezraimg from "images/avatar/02.png";
-import webthreeimg from "images/avatar/03.png";
-import boxydudeimg from "images/avatar/04.png";
-import quantumcomputingimg from "images/avatar/05.png";
-import sathoshiimg from "images/avatar/06.png";
-import sonicimg from "images/avatar/07.png";
 import { X_Handle } from "src/constants";
 
 export const DEFAULT_AGENT_ID = "2";
```

Comment:
can we delete this too, don't think we use it anywhere

Reply:
Which line?

Reply:
Removed DEFAULT_AGENT_ID